### ISSUE TITLE
Improve Readme.md file

### ISSUE THREAD

**Issue by WinXprt** [prolsen/barff#1]
Can you improvise the Readme.md file for better understandability?

**Comment by prolsen** [comment]
Yes, I will try finding some time.  I just moved all my code over from a different github account so I need to clean house a bit.
